dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -91,7 +556,7 @@ data:
               "showLineNumbers": false,
               "showMiniMap": false
             },
-            "content": "<h1><center>5xx errors rate for service providers less than 1%</center></h1>",
```

Comment:
IMHO everything here and below shouldn't be changed, except grid positions

Reply:
agreed. new content there now that reflects this.